README.md

```diff
@@ -129,6 +129,12 @@ Tusky can also use [UnifiedPush](https://unifiedpush.org/) to show notifications
 
 Google Services don't work on all devices and we want to give users the freedom to chose their push provider.
 
+## Profiles
+
+### Why do I see a "Information below may reflect the user's profile incompletely. Press to open full profile in browser." message on user's profiles?
+
+This happens when you are viewing profiles on servers other than your homeserver. Your homeserver can't know all posts of all users in the fediverse and Tusky can only show you posts it gets from your homeserver. Clicking on the message will take you to this other server and you will be able to see all public posts of the user there.
```

Comment:
Not "home server" - or is that a know parlance of fediverse?

Fediverse (large starting letter)?